mmv1/products/compute/NodeTemplate.yaml

```diff
@@ -175,3 +179,23 @@ properties:
     enum_values:
       - 'ENABLED'
       - 'NONE'
+  - name: 'disks'
+    type: Array
+    description: |
+      List of the type, size and count of disks attached to the
+      node template
+    item_type:
+      type: NestedObject
+      properties:
```

Comment:
Just want to double check if these three properties `diskCount`, `diskType` and `diskSizeGb` are required or optional?

Reply:
Adding a pool of local ssds is completely optional.